cmd/kueuectl/app/list/list_clusterqueue_test.go

```diff
@@ -0,0 +1,153 @@
+				Client: fake.NewSimpleClientset(
+					utiltesting.MakeClusterQueue("cq1").
+						Condition(v1beta1.ClusterQueueActive, metav1.ConditionTrue, "", "").
+						Cohort("cohort1").
+						Creation(testStartTime.Add(-time.Hour).Truncate(time.Second)).
+						Obj(),
+					utiltesting.MakeClusterQueue("cq2").
+						Condition(v1beta1.ClusterQueueActive, metav1.ConditionFalse, "", "").
+						PendingWorkloads(1).
+						Creation(testStartTime.Add(-time.Hour).Truncate(time.Second)).
+						Obj(),
+					utiltesting.MakeClusterQueue("cq3").
+						Condition(v1beta1.ClusterQueueActive, metav1.ConditionUnknown, "", "").
+						AdmittedWorkloads(1).
+						Creation(testStartTime.Add(-time.Hour).Truncate(time.Second)).
+						Obj(),
+					utiltesting.MakeClusterQueue("cq4").
+						Condition("other", metav1.ConditionTrue, "", "").
+						Creation(testStartTime.Add(-time.Hour).Truncate(time.Second)).
+						Obj(),
+				).KueueV1beta1(),
+			},
+			wantOut: []string{
+				"NAME   COHORT    PENDING WORKLOADS   ADMITTED WORKLOADS   ACTIVE   AGE",
+				"cq1    cohort1   0                   0                    true     60m",
+				"cq2              1                   0                    false    60m",
+				"cq3              0                   1                    false    60m",
+				"cq4              0                   0                    false    60m\n",
+			},
+		},
+		"should print active cluster queue list": {
+			o: &ClusterQueueOptions{
+				PrintObj: printClusterQueueTable,
+				Active:   "true",
+				Client: fake.NewSimpleClientset(
+					utiltesting.MakeClusterQueue("cq1").
+						Condition(v1beta1.ClusterQueueActive, metav1.ConditionTrue, "", "").
+						Creation(testStartTime.Add(-time.Hour).Truncate(time.Second)).
+						Obj(),
+					utiltesting.MakeClusterQueue("cq2").
+						Condition(v1beta1.ClusterQueueActive, metav1.ConditionFalse, "", "").
+						Creation(testStartTime.Add(-time.Hour).Truncate(time.Second)).
+						Obj(),
+					utiltesting.MakeClusterQueue("cq3").
+						Condition(v1beta1.ClusterQueueActive, metav1.ConditionUnknown, "", "").
+						Creation(testStartTime.Add(-time.Hour).Truncate(time.Second)).
+						Obj(),
+					utiltesting.MakeClusterQueue("cq4").
+						Condition("other", metav1.ConditionTrue, "", "").
+						Creation(testStartTime.Add(-time.Hour).Truncate(time.Second)).
+						Obj(),
+				).KueueV1beta1(),
+			},
+			wantOut: []string{
+				"NAME   COHORT   PENDING WORKLOADS   ADMITTED WORKLOADS   ACTIVE   AGE",
+				"cq1             0                   0                    true     60m\n",
+			},
+		},
+	}
+	for name, tc := range testCases {
+		t.Run(name, func(t *testing.T) {
+			s, _, out, _ := genericiooptions.NewTestIOStreams()
+			tc.o.IOStreams = s
+
+			gotErr := tc.o.Run()
+			if diff := cmp.Diff(tc.wantErr, gotErr, cmpopts.EquateErrors()); diff != "" {
+				t.Errorf("Unexpected error (-want/+got)\n%s", diff)
+			}
+
+			gotOut := out.String()
+			joinedWantOut := strings.Join(tc.wantOut, "\n")
```

Comment:
why not split the gotOut, instead, maybe we don't need the special "\n" at the end of the lat item